automation/prepare-release.py

```diff
@@ -40,6 +40,16 @@
 
 ensure_working_tree_clean()
 
+step_msg(f"Calculating remote owner {remote}")
+result = run_cmd_checked(["git", "remote", "get-url", remote], stdout=subprocess.PIPE)
+repo_url = result.stdout.decode('utf8').strip()
+print(repo_url)
```

Comment:
super nit: something like: `f"Remote repo url is: {repo_url}"` can help in case someone's URL is off